src/components/navigation/header/ComparisonItem.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { useI18n } from '@smg-automotive/i18n-pkg';
+import { chakra } from '@chakra-ui/react';
+
+import { CustomEvent, navigationEventCategory } from 'src/types/tracking';
+import { CompareIcon } from 'src/components/icons';
+import Count from 'src/components/count';
+import Box from 'src/components/box';
+import Badge from 'src/components/badge';
+
+export const shouldShowComparisonLink = (
+  comparisonItemIds?: number[] | null,
+): comparisonItemIds is number[] => {
+  return !!comparisonItemIds && Array.isArray(comparisonItemIds);
+};
+
+export const getComparisonUrl = (comparisonItemIds: number[]) => {
+  const baseUrl = 'comparison';
+  if (comparisonItemIds.length === 0) return baseUrl;
+  return `${baseUrl}/${comparisonItemIds.join('/')}`;
+};
+
+type Props = {
+  comparisonItemIds?: number[] | null;
+  trackEvent?: (event: CustomEvent) => void;
+};
+
+const ComparisonItem: FC<Props> = ({ comparisonItemIds, trackEvent }) => {
+  const { t, language } = useI18n();
+
+  return (
+    <chakra.a
+      position="relative"
+      href={`/${language}/${getComparisonUrl(comparisonItemIds ?? [])}`}
```

Comment:
maybe... 🤔  isn't that so that non-replatformed pages work on branches? Because it confused me quite a few times already that a click on a link in the header goes to INT on branch deployments

Reply:
The general use case is if your project does not run under the root/www domain. (Not sure I  think it ended up not working in the react projects we use internally) but we also had external partners using the header in the past

Reply:
Done ✅ 
I find it quite annoying that with this change, comparison URL opens on INT on localhost. I created a task to improve this: https://smg-au.atlassian.net/browse/DM-3415

Since a lot of pages are now replatformed, we should improve this